src/js/components/Pagination/StyledPageControl.js

```diff
@@ -2,6 +2,43 @@ import styled from 'styled-components';
 
 import { Button } from '../Button';
 
+const BUTTON_STATES = ['default', 'active', 'disabled', 'hover', 'selected'];
+const defaultStyles = {
+  border: {
+    radius: '4px',
+  },
+  pad: '4px',
+};
+
+// Assemble button theme object for each button state
+const buildButtonStyle = (buttonTheme, kind) => {
```

Comment:
since `buttonTheme` doesn't seem to be a value that changes. what if it's just a const in this function and the only thing passed to buildButtonStyle is `kind`?

Reply:
Discussed offline. Theme context is needed, so needs to be passed into this function.